### ISSUE TITLE
Accessibility refinement – aria-expanded attribute on container element

### ISSUE THREAD

**Issue by simonminter** [tomickigrzegorz/show-more#75]
Working on a project that included `showmore.js`, automated accessibility tests (carried out with Axe) were failing, with the "Elements must only use allowed ARIA attributes" issue.

This seemed to be due to the `aria-expanded` attribute being applied to the showmore container element _as well as_ the showmore-generated button element. Looking at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded, the `aria-expanded` attribute should be added to the button element, which is happening correctly, but it does not need to be added to the container element.

The addition of `aria-expanded` on the container is used by code to affect behaviour, so if it is replaced with a different/custom attribute, the functionality will continue to work, without triggering an accessibility test failure.

**Comment by simonminter** [comment]
Pull request for a fix is here: https://github.com/tomickigrzegorz/show-more/pull/76
